kubernetes/overlays/backend-for-frontend/test/configmap.yaml

```diff
@@ -62,3 +66,5 @@ data:
   EXPORT_ESERVICE_PATH: "{{NAMESPACE}}/eservices-export"
   PRESIGNED_URL_GET_DURATION_MINUTES: "{{PRESIGNED_URL_GET_DURATION_MINUTES}}"
   PRESIGNED_URL_PUT_DURATION_MINUTES: "{{PRESIGNED_URL_PUT_DURATION_MINUTES}}"
+
+  CLIENT_ASSERTION_AUDIENCE: "{{CLIENT_ASSERTION_JWT_AUDIENCE}}"
```

Comment:
are this and [this](https://github.com/pagopa/pdnd-interop-platform-deployment/pull/624/files#diff-77200e323a98461ef0c64a6f97fd8e61a9bb8b87e579eba17db2742208a93fafR13) both required?

Reply:
No it's a wrong repetition.
Thanks